### ISSUE TITLE
www.defibrillateurs.info géré par afpaic.fr, code provenant de info-flash.com

### ISSUE THREAD

**Issue by lefuturiste** [osm-fr/attributions#400]
### Ce signalement concerne…

La carte principale de : https://www.defibrillateurs.org/

Signalé par https://www.openstreetmap.org/user/benoitdd

### Présence de l'attribution

❌ Pas d'attribution

### (Facultatif) Quel est le fond de carte ?

openstreetmap.org

### Prise de contact

❌ Je ne l'ai pas contacté

### Informations supplémentaires

- code en question : https://www.info-flash.com/components/com_infoflash/assets/map/map.js

- Responsable contacté le : 
  - Responsable contacté par (courriel, Twitter…) : 
- Lié au signalement : #

**Comment by veterini-osm** [comment]
Je viens de leur envoyer l'e-mail type le 25/05/2025

**Comment by Jean-BaptisteC** [comment]
Le site est https://www.defibrillateurs.info/

**Comment by veterini-osm** [comment]
Effectivement, j'avais bien envoyé le mail à l'adresse de contact sur [https://www.defibrillateurs.info/](https://www.defibrillateurs.info/) (j'étais initialement tombé sur #515). Pas de réponse pour le moment.
